robottelo/datafactory.py

```diff
@@ -179,9 +179,9 @@ def valid_emails_list():
             gen_string('alphanumeric'),
             gen_string('alphanumeric'),
         ),
-        u'"():;"@example.com',
         u'!#$%&*+-/=?^`{|}~@example.com',
     ]
+    return [email.replace('"', '').replace('`', '') for email in email_list]
```

Comment:
I think there is no point having them, but I can't force gen_string not to create them, or can I? Basically, I just thought that if something's making trouble, it shouldn't be on the valid data list, so that tests don't have to tweak data afterwards to make them actually valid.